src/main/assembly/dist/cfg/application.properties

```diff
@@ -4,7 +4,7 @@ deposits.permissions=rwxrwx---
 tempdir=/var/opt/dans.knaw.nl/tmp/easy-sword2
 # Require at least some 2 G  spare disk space
 tempdir.margin-available-diskspace-mb=2000
-base-url=http://yourdomain-here.org/
+base-url=http://test.dans.knaw.nl/
```

Comment:
mmm, maybe you're right. I was thinking the 'reasonable default' should cater towards the local vm. Also I don't expect anyone else to use this software, other than us...